### PR TITLE
Nathan downs event level data docs

### PR DESCRIPTION
**Pull request by nathan-downs**
## Test plan

Visual testing of handbook additions.

### CODE REVIEW

**Comment by nathan-downs** [comment]
@cla-bot check

File: doc/dev/background-information/adding_event_level_data.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Adding, changing, and debugging user event data
+
+This document outlines the process for adding or changing the raw user event data collected from Sourcegraph instances. This is limited to certain managed instances (cloud) where the customer has signed a corresponding data collection agreement.
+
+### User event data philosophy
+
+[Raw user event data](https://docs.sourcegraph.com/dev/background-information/data-usage-pipeline) is collected from logs in the event_logs table in the instance primary database and sent to Sourcegraph centralized analytics. These [events](https://sourcegraph.sourcegraph.com/search?q=context:global+repo:%5Egithub%5C.com/sourcegraph/sourcegraph%24+file:internal/database/event_logs.go+Event+type:symbol+select:symbol.struct&patternType=standard) are a product of events performed by users or the system and represent our customers’ most sensitive data. We must preserve and build trust through only careful additions and changes to events added to the egress pipeline.
```
Comment (akalia25) [inline doc/dev/background-information/adding_event_level_data.md:7]:
```suggestion
... is collected from logs in the `event_logs` table ...
```
Nitpick, since event_logs is a table maybe we can format it differently

File: doc/dev/background-information/adding_event_level_data.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+2. In the [raw table](https://console.cloud.google.com/bigquery?project=telligentsourcegraph&ws=!1m5!1m4!4m3!1stelligentsourcegraph!2sdotcom_events!3sevents_usage_raw), the subscription unloads any new data into this table as an array of JSON objects. The payloads are capped at source to 10MB.
+3. In the [subscription](https://console.cloud.google.com/cloudpubsub/subscription/detail/dotcom-events-usage?project=telligentsourcegraph), there are metrics available in the cloud console. There is not currently any automated snapshot generation.
+4. In the [pub/sub topic](https://console.cloud.google.com/cloudpubsub/topic/detail/dotcom-events-usage?project=telligentsourcegraph), there are also metrics available in the cloud console. There is not currently any automated snapshot generation. There is currently a 7 day retention duration for messages in the queue.
+5. In the instance–at this point it would point to an issue with the instance sending data at all. Check if there is also an interruption in [ping](https://docs.sourcegraph.com/dev/background-information/adding_ping_data) or other telemetry data.
```
Comment (akalia25) [inline doc/dev/background-information/adding_event_level_data.md:62]:
```suggestion
5. In the instance, this would point to an issue with the instance sending data at all. Check if there is also an interruption in [ping](https://docs.sourcegraph.com/dev/background-information/adding_ping_data) or other telemetry data.
```
nitpick, the wording with "instance-at" and the "this point it would point" maybe this a little hard to follow

File: doc/dev/background-information/adding_event_level_data.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+This document outlines the process for adding or changing the raw user event data collected from Sourcegraph instances. This is limited to certain managed instances (cloud) where the customer has signed a corresponding data collection agreement.
+
+### User event data philosophy
+
+[Raw user event data](https://docs.sourcegraph.com/dev/background-information/data-usage-pipeline) is collected from logs in the `event_logs` table in the instance primary database and sent to Sourcegraph centralized analytics. These [events](https://sourcegraph.sourcegraph.com/search?q=context:global+repo:%5Egithub%5C.com/sourcegraph/sourcegraph%24+file:internal/database/event_logs.go+Event+type:symbol+select:symbol.struct&patternType=standard) are a product of events performed by users or the system and represent our customers’ most sensitive data. We must preserve and build trust through only careful additions and changes to events added to the egress pipeline.
+
+All user event data must be:
+
+Anonymous (with only one exception–the email address of the initial site installer)
```
Comment (akalia25) [inline doc/dev/background-information/adding_event_level_data.md:11]:
```suggestion
Anonymous (with only one exception, the email address of the initial site installer)
```
With the hyphen it seemed like "exception-the" was one thing, this might just be a me issue so feel free to discard this comment

File: doc/dev/background-information/adding_event_level_data.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+Non-specific (e.g., no repo names, no usernames, no file names, no specific search queries, etc.)
```
Comment (akalia25) [inline doc/dev/background-information/adding_event_level_data.md:12]:
```suggestion
All user event data must be:
1. Anonymous (with only one exception–the email address of the initial site installer)
2. Non-specific (e.g., no repo names, no usernames, no file names, no specific search queries, etc.)
```
Since it ends with a colon (":") maybe we can make this into a list ? (aesthetic change, feel free to dismiss)

**Review by akalia25 — APPROVED** [review]
Looks good to me! The docs were very clear and easy to follow. I like how it was formatted and how detailed the debugging section was :)
